site/src/github.rs

```diff
@@ -636,46 +626,65 @@ async fn summarize_run(ctxt: &SiteCtxt, commit: QueuedCommit, is_master_commit:
     } else {
         String::new()
     };
-    let (primary, secondary) = comparison.summarize_by_category(&benchmark_map);
+    let (inst_primary, inst_secondary) = inst_comparison
+        .clone()
+        .summarize_by_category(&benchmark_map);
+
+    let mut table_written = false;
+    let stats = vec![
+        (
+            "Instruction count",
+            Stat::Instructions,
+            false,
+            inst_comparison,
+        ),
+        (
+            "Max RSS (memory usage)",
+            Stat::MaxRSS,
+            true,
+            calculate_stat_comparison(ctxt, &commit, Stat::MaxRSS).await?,
```

Comment:
nit: you could also set these to `None`, and in the loop check if there `None` and if they are run `calculate_stat_comparison` using the provided `stat`

Reply:
It came to mind, but it seemed more "dirty" to me to do it with such a "mutable" way :) I can change it if you want.